README.md

```diff
@@ -1,16 +1,26 @@
-SimpleLogin - privacy-first email alias and Single Sign-On (SSO) Identity Provider
+SimpleLogin - One of the first privacy respecting Email Aliasing and Single Sign-On (SSO) Identity Service(s).
 ---
 
-https://simplelogin.io
+# Bio
+This repository contains the everything related to the website of SimpleLogin SAS.
 
-This repo contains the code behind the main website at https://www.simplelogin.io
+# Online Presence
+[Website](https://simplelogin.io)
+[Twitter](https://twitter.com/simple_login)
+[Reddit](https://www.reddit.com/r/Simplelogin/)
+[Producthunt](https://www.producthunt.com/products/simplelogin)
+[Other](#)
 
-It's developed using https://gohugo.io, a static site generator.
+# Credit
+This website is developed using a pice of free and open source (FOSS) software called [hugo](https://gohugo.io), a static website generator for all.
```

Comment:
Hugo is indeed an important part of the website but I don't think we need to advertise their technology :)

Reply:
It's not an AD, more an "let's give them credit to avoid trouble type of thing".

Reply:
@nguyenkims removed it as per your wishes.